lnrpc/rest-annotations.yaml

```diff
@@ -133,6 +133,9 @@ http:
     - selector: lnrpc.Lightning.BakeMacaroon
       post: "/v1/macaroon"
       body: "*"
+    - selector: lnrpc.Lightning.ListMacaroonIDs
+      post: "/v1/macaroon/list"
```

Comment:
Should be `get`. And maybe `v1/macaroon/ids`?

Reply:
fixed